src/main/java/ua/cv/tim/service/impl/Parser.java

```diff
@@ -0,0 +1,90 @@
+package ua.cv.tim.service.impl;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.htmlunit.HtmlUnitDriver;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.security.core.context.SecurityContextHolder;
+import org.springframework.security.core.userdetails.UserDetails;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+import ua.cv.tim.model.Army;
+import ua.cv.tim.model.Player;
+import ua.cv.tim.model.User;
+import ua.cv.tim.model.Village;
+import ua.cv.tim.service.PlayerService;
+import ua.cv.tim.service.UserService;
+import ua.cv.tim.service.VillageService;
+
+import java.util.List;
+
+/**
+ * Created by Serhii Starovoit on 1/25/2017 in 1:25 PM.
+ */
+
+@Service(value = "parser")
+public class Parser {
+
+    @Autowired
+    VillageParser villageParser;
+
+    @Autowired
+    ArmyParser armyParser;
+
+    @Autowired
+    VillageService villageService;
+
+    @Autowired
+    UserService userService;
+
+    @Autowired
+    PlayerService playerService;
+
+    @Transactional
```

Comment:
If you use @Transactional over all methods better put it above class declaration and don't use with methods